apps/src/netsim/netsim.js

```diff
@@ -332,7 +334,6 @@ NetSim.prototype.setRouterCreationTime = function (creationTimestampMs) {
  * @param {number} newBandwidth in bits/second
  */
 NetSim.prototype.setRouterBandwidth = function (newBandwidth) {
-  this.routerBandwidth_ = newBandwidth;
```

Comment:
Yeah, RubyMine was telling me they had no usages. Thanks for catching the missed setter; I still find it a little surprising that JSHint doesn't catch that.